pkgs/by-name/su/surge-XT/package.nix

```
@@ -66,11 +76,23 @@ stdenv.mkDerivation rec {
     ]
   );
 
-  # see https://github.com/NixOS/nixpkgs/pull/149487#issuecomment-991747333
+  patches = [ ./clap-option.diff ];
```

Could you put this closer to `src`? It's a slightly personal preference of mine but I'm sure you'd agree that patches are closely tied to the source, so diffs to both of which usually come together. Also, to make the future diffs smaller:

```suggestion
  patches = [
    ./clap-option.diff
  ];
```

---

Yes, [it's merged](https://github.com/surge-synthesizer/surge/pull/8202)